package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "description": "An enterprise-class UI components based on Ant Design and Angular",
   "engines": {
-    "node": "^14.15.5 || ^16.10.0"
+    "node": "^14.20.0 || ^18.10.0"
```

Comment:
Angular 15 supports node version: `14.20.x, 16.13.x and 18.10.x` [official doc](https://angular.io/guide/update-to-version-15#angular-v15-supports-nodejs-versions-1420x-1613x-and-1810x)
Or we can set it be with `>=14.0.0 <17.0.0` just like [material](https://github.com/angular/components/blob/main/package.json#L12)?

Reply:
> Angular 15 supports node version: `14.20.x, 16.13.x and 18.10.x` [official doc](https://angular.io/guide/update-to-version-15#angular-v15-supports-nodejs-versions-1420x-1613x-and-1810x) Or we can set it be with `>=14.0.0 <17.0.0` just like [material](https://github.com/angular/components/blob/main/package.json#L12)?

we can also set it like angular: ```^14.20.0 || ^16.13.0 || >=18.10.0"``` what do you think ?

Reply:
hey, i authorize same node engine as Angular did [here](https://github.com/angular/angular/blob/main/package.json)